requirements/edx/pre.txt

```diff
@@ -4,6 +4,9 @@
 #   * @mollydb - to check licensing
 #   * One of @e0d, @jarv, or @feanil - to check system requirements
 
+# Use a modern setuptools instead of distribute
+setuptools==9.1
```

Comment:
As mentioned in the commit message, nltk==2.0.4 is not compatible with the most recent versions of setuptools.